### PR TITLE
0.6.0 beta.4

### PR DESCRIPTION
**Pull request by joswarmer**
Done:
- Update to newest serialization format
- JSON files for M3 directly copied from lionweb/organisation
- Improved usage of optional element in editor

### CODE REVIEW

File: modelstore/lioncore/LionCore.M3.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-   "serializationFormatVersion": "2023.1",
+   "serializationFormatVersion": "1",
```
Comment (dslmeinte) [inline modelstore/lioncore/LionCore.M3.json:2]: _Should_ be `"2023.1"`, right? (Same for next JSON file.)

File: src/picode/commandline/LionWeb2FreonTemplate.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { Concept, ConceptInterface, Containment, Enumeration, Feature, Language, PrimitiveType, Property, Reference } from "../language/gen/index";
+
+export class LionWeb2FreonTemplate {
+
+    generateFreonAst(metamodel: FreModelUnit): string {
+        let result = "";
+        result += (`language ${metamodel.name}\n`);
+        (metamodel as Language).entities.forEach(entity => {
+            switch(entity.freLanguageConcept()) {
+                case "Concept":
```
Comment (dslmeinte) [inline src/picode/commandline/LionWeb2FreonTemplate.ts:11]: (I guess a case for `Annotation` is missing here.)

Comment (joswarmer) [reply src/picode/commandline/LionWeb2FreonTemplate.ts:11]: yes, this code is WIP

Comment (joswarmer) [reply src/picode/commandline/LionWeb2FreonTemplate.ts:11]: Changed it

Comment (joswarmer) [reply src/picode/commandline/LionWeb2FreonTemplate.ts:11]: I need to addd Annotation support in Freon first :-(.

**Pull request merged**